lc-hax/Scripts/Modules/Possession/Controllers/IEnemyController.cs

```diff
@@ -29,9 +46,21 @@ internal interface IEnemyController<T> : IController where T : EnemyAI {
 
     bool IsAbleToMove(T enemyInstance) => true;
 
+    bool IsAbleToRotate(T enemyInstance) => true;
+
+    bool CanUseEntranceDoors(T enemyInstance) => true;
+
     string? GetPrimarySkillName(T enemyInstance) => null;
 
     string? GetSecondarySkillName(T enemyInstance) => null;
+    float? InteractRange(T enemyInstance) => 2.5f;
```

Comment:
If most enemies have an interact range of `1.0f` why not just make that as default?

Reply:
only the small ones,  the big ones needs to be larger